.travis.yml

```diff
@@ -15,6 +11,20 @@ php:
 git:
   depth: 1
 
+compiler:
```

Comment:
Does it work this way? Last time I checked, Travis ignored "compiler" if language: php is set